client/src/components/ContentNavigator/index.ts

```diff
@@ -20,11 +20,13 @@ import { ContentItem } from "./types";
 import { getUri, isContainer as getIsContainer } from "./utils";
 
 const fileValidator = (value: string): string | null =>
-  /^([a-zA-Z0-9\s._-]+)\.\w+$/.test(value)
+  /^([^,/<>?;:\\{}|`=+*&^%#@!~\uff01-\uff5e\u3000-\u303f\u0080-\u00ff]+)\.\w+$/.test(
```

Comment:
This is a little hard to read, but I think I understand. A valid file contains any character _except_ those in the list above, right? 

In any case, do you mind adding a comment in code about what this regex does?

Reply:
Done